osu.Game/Screens/Menu/MenuSideFlashes.cs

```diff
@@ -102,7 +102,7 @@ protected override void OnNewBeat(int beatIndex, TimingControlPoint timingPoint,
 
         private void flash(Drawable d, double beatLength, bool kiai, TrackAmplitudes amplitudes)
         {
-            d.FadeTo(Math.Max(0, ((d.Equals(leftBox) ? amplitudes.LeftChannel : amplitudes.RightChannel) - amplitude_dead_zone) / (kiai ? kiai_multiplier : alpha_multiplier)), box_fade_in_time)
+            d.FadeTo(System.Math.Max(0, ((d.Equals(leftBox) ? amplitudes.LeftChannel : amplitudes.RightChannel) - amplitude_dead_zone) / (kiai ? kiai_multiplier : alpha_multiplier)), box_fade_in_time)
```

Comment:
Not sure about this

Reply:
using System for one time

Reply:
We suggest move all the usings in using section, even they are used once

Reply:
Ok